### PR TITLE
Add mdfile to INPUT

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
NOT only USE_MDFILE_AS_MAINPAGE needs to be set, but also the MD file needs to be added to INPUT as mentioned in https://stackoverflow.com/a/13442157

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
ping ;)

File: src/rosdoc_lite/doxygenator.py [hunk 1]
```diff
@@ -194,7 +198,8 @@ def package_doxygen_template(template, rd_config, path, package, html_dir, heade
               '$HTML_HEADER': header_filename,
               '$HTML_OUTPUT': os.path.realpath(html_dir),
               '$IMAGE_PATH': rd_config.get('image_path', path), #default to $INPUT
-              '$INPUT':  path, '$PROJECT_NAME': package,
+              '$INPUT': ' '.join([path, mdfile]),
```
Comment (tfoote) [inline src/rosdoc_lite/doxygenator.py]: It looks like this is going to leave trailing whitespace. Have you verified that doesn't create any warnings etc?

Comment (MatthijsBurgh) [reply src/rosdoc_lite/doxygenator.py]: The trailing space doesn't cause any trouble and/or warnings.

**Comment by MatthijsBurgh** [comment]
Doxygen provides the option to use a Markdown file as the mainpage, http://www.doxygen.nl/manual/config.html#cfg_use_mdfile_as_mainpage. If this option is not provided the mainpage will be empty. As also mentioned in the Doxygen documentation, the mdfile should be part of the `input`.

**Pull request merged**